packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.internal.js

```diff
@@ -250,9 +250,7 @@ describe('ChangeEventPlugin', () => {
     input.checked = true;
     // Under the hood, uncheck the box so that the click will "check" it again.
     setUntrackedChecked.call(input, false);
-    input.dispatchEvent(
-      new MouseEvent('click', {bubbles: true, cancelable: true}),
-    );
+    input.click();
```

Comment:
Why did these events need to be changed?

Reply:
JSDOM no longer triggers default actions on dispatching an event it seems. There's no spec around that as far as I know. In any case `.click()` works as well in a browser: https://codesandbox.io/s/should-track-radio-button-cousins-in-a-group-thb8k